Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -644,39 +654,40 @@ IPCCommandResult ES::ExportContentBegin(Context& context, const IOCtlVRequest& r
 
 ReturnCode ES::ExportContentData(Context& context, u32 content_fd, u8* data, u32 data_size)
 {
-  const auto iterator = context.title_export.contents.find(content_fd);
-  if (!context.title_export.valid || iterator == context.title_export.contents.end() ||
-      iterator->second.content.m_position >= iterator->second.content.m_content.size)
+  ReturnCode ret = ES_EINVAL;
```

Comment:
Yes. That was needed in the old code because we used to read contents ourselves and check bounds manually, but this is now all handled by ReadContent, which will even make sure the proper error code is returned in several other error cases.